base/client.jl

```diff
@@ -253,7 +253,7 @@ function process_options(opts::JLOptions)
     repl                  = true
     startup               = (opts.startupfile != 2)
     history_file          = (opts.historyfile != 0)
-    quiet                 = (opts.quiet != 0)
+    banner                = (opts.banner == 0)
```

Comment:
This should read as either `banner = (opts.banner != 0)` or `quiet = (opts.banner == 0)`.